### PR TITLE
Add project codespaces config

### PR DESCRIPTION
**Pull request by lonrun**

### CODE REVIEW

File: tutorail/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## CodeQuery使用教程
```
Comment (ValKmjolnir) [inline tutorail/README.md:1]: You have a typo in the name of the directory, expected `tutorial`, not `tutorail`

**Review by ValKmjolnir — CHANGES_REQUESTED** [review]

File: tutorail/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### 使用Github Codespaces 来体验CodeQuery分析能力 
+
+#### 步骤
+
+- 在项目主页切换到目标分支
```
Comment (zhouang777) [inline tutorail/README.md:7]: 不用写切换分支，main分支理应支持

File: tutorail/README.md (hunk 1 above)
Comment (lonrun) [reply tutorail/README.md]: ok

**Review by ValKmjolnir — APPROVED** [review]

File: tutorail/README.md (hunk 2 above)
Comment (lonrun) [reply tutorail/README.md]: @FunJim  please update this in the following pr

**Pull request merged**